gulpfile.js

```diff
@@ -169,61 +155,74 @@ function build(options) {
       removePragmas: options.removePragmas,
     }),
     createGalleryList(noDevelopmentGallery),
-    buildSpecs()
-  ).then(() => {
-    return copyAssets(outputDirectory);
-  });
+    buildSpecs(),
+  ]);
+
+  return copyAssets(outputDirectory);
 }
 
-gulp.task("build", function () {
-  const argv = yargs.argv;
+export function build() {
   const minify = argv.minify ? argv.minify : false;
   const removePragmas = argv.pragmas ? argv.pragmas : false;
   const sourcemap = argv.sourcemap ? argv.sourcemap : true;
   const node = argv.node ? argv.node : true;
 
-  return build({
+  return buildCesium({
     minify: minify,
     removePragmas: removePragmas,
     sourcemap: sourcemap,
     node: node,
   });
-});
+}
+export default build;
```

Comment:
Is this export needed?

Reply:
Yes, this becomes the [default gulp task](https://gulpjs.com/docs/en/getting-started/creating-tasks/).